### ISSUE TITLE
Playlist: Add an option for playback speed to 2.5x

### ISSUE THREAD

**Issue by anthonypkeane** [brave/brave-browser#29623]
## Description 
https://github.com/brave/brave-ios/issues/6682

https://twitter.com/vac93302099/status/1608950353764175872?s=20&t=h_jqc-7IWDUd4RNfelvq2w

Please add an option for playback speed to 2.5x

**Comment by bsclifton** [comment]
Closing as `wontfix` for now. I think we need a more comprehensive plan for Playlist. We can revisit specific features once we find a way to land the existing code.
